PWGHF/HFC/DataModel/CorrelationTables.h

```diff
@@ -470,6 +470,30 @@ DECLARE_SOA_TABLE(DmesonSelection, "AOD", "DINCOLL", // Selection of D meson in
                   aod::hf_selection_dmeson_collision::DmesonSel);
 
 // Note: definition of columns and tables for Electron Hadron correlation pairs
+// definition of columns and tables for electron properties
+namespace hf_electron
+{
+DECLARE_SOA_COLUMN(Phi, phi, float);                   //! Phi of electron
+DECLARE_SOA_COLUMN(Eta, eta, float);                   //! Eta of electron
+DECLARE_SOA_COLUMN(PtElectron, ptElectron, float);     //! Transverse momentum of electron
+DECLARE_SOA_COLUMN(IsLSElectron, isLSElectron, int);   //! like sign Electron
+DECLARE_SOA_COLUMN(IsULSElectron, isULSElectron, int); //! unLike sign Electron
```

Comment:
Is this needed to be an `int`? Isn't a `bool` enough? Probably I did not get what this column is (similarly for the `IsLSElectron`)

Reply:
This is very misleading. Please fix the naming.